### ISSUE TITLE
Crap load of errors, please help.

### ISSUE THREAD

**Issue by marcusx2** [Ryan-rsm-McKenzie/CommonLibSSE#44]
I am using the latest visual studio. I tried to follow your tutorial, but I got a lot of errors. Like, a lot. Please take a look:
![errors](https://user-images.githubusercontent.com/5656275/98411023-12390300-2054-11eb-8095-450063010201.png)

It's like the whole skse code is broken for some reason. I made the skse a .lib, added to the linker, added the directory to the header files. So my plugin file doesn't accuse any error, it's the skse itself that has tons of errors. Maybe I missed something on the tutorial?

**Comment by marcusx2** [comment]
I am able to build the skse project without errors and generate the .lib. The problem is when I try to build my plugin project, then suddently all sorts of errors pop up on skse. Which is confusing because building skse alone doesn't cause any issues.

**Comment by marcusx2** [comment]
@Ryan-rsm-McKenzie I also tried with visual studio 2015, same result. Please help man, it's probably something very stupid but I can't figure it out. It's like, yeah, I can build skse, generate the .lib. Then I make my plugin that import the skse headers, and all these errors pop up. Not on my .cpp file, mind you. They pop up on skse. How can I build skse just fine but when I try to use it by importing the headers it blows up?

**Comment by marcusx2** [comment]
I **almost** fixed everything. I hade to:

1. `#include <stdio.h>` to IDebugLog.h.
2. `#include <string>` to GameForms.h and Utilities.h.
3. `#include <windows.h>`  to GameTypes.h
4. Remove static asserts(couldn't ignore them like the tutorials asks on visual studio 2015. I switched VS version because I thought it would solve my problem but it didn't).

Now I just have these:
![image](https://user-images.githubusercontent.com/5656275/98448045-24747900-2108-11eb-97f0-31117f280e75.png)

**Comment by marcusx2** [comment]
Ok I fixed everything. Now I need to add the references to the project:
![image](https://user-images.githubusercontent.com/5656275/98448559-95696000-210b-11eb-8e0b-705fca2716d9.png)

**Comment by fireundubh** [comment]
You basically did everything wrong and missed steps in the tutorial.

**Comment by marcusx2** [comment]
Hmm...I see, it's because I don't use CommonLibSSE, but I was redirected to the Ryan's tutorial for the part of building a skse plugin. So I followed all until
![image](https://user-images.githubusercontent.com/5656275/98452779-125b0080-2131-11eb-88db-41c4b806d23b.png)
and didn't follow the CommonLibSSE part because I thought it was specific to CommonLibSSE only, not general skse plugins.

That makes sense I guess, thanks. I got it working without CommonLibSSE anyways.
